README.md

```diff
@@ -13,4 +13,101 @@ Read our [Contribution Guidelines](CONTRIBUTING.md) before you contribute.
 
 ## List of Algorithms
 
-See our [directory](DIRECTORY.md).
+
```

Comment:
My sense is that we should continue to generate `DIRECTORY.md` like other TheAlgorithms repos.

Reply:
I guess it could be fine to have the list in the `README.md` file. Based on our discussion in discord with @siriak, we decided to have the list of algorithms inside the `README.md` because it's more accessible for readers, and they could see the list easily,
And about the changes in my last commit, I didn't add all list like it was in `DIRECTORY,md`, before we had linked to all files, but in `README.md` I makes them smaller instead to mention to all files I just linked them to their folder not all files inside the folder.

Maybe it could be fine to more talk about it, @siriak what do you think?

Reply:
We cannot generate it any longer because there're no word separators in filenames (e.g. binarysearch.go contains no separator between "binary" and "search").

Reply:
What about a .json file that contains a map of human name to Go compliant name.  I could modify the GitHub Action to process that file.

Reply:
We'll have to maintain that file instead of readme. And also there are cases when there are multiple implementations of the same algorithms (somealgorithm1.go, somealgorithm2.go, etc.). We probably don't want to add all of them to readme, only one entry to directory with all of them.

Reply:
Why do you want to generate it automatically? It was first done for the website, but now it's not required.